components/launcher/src/main.rs

```diff
@@ -32,6 +33,10 @@ fn main() {
             error!("Launcher exiting with 1 due to err: {}", err);
             process::exit(1);
         }
+        Ok(0) => {
+            info!("Launcher exiting with code 0");
+            process::exit(0);
+        }
```

Comment:
I like the change to make a zero exit status an info-level log. I think we can make use of [the `log!` macro](https://docs.rs/log/0.4.6/log/macro.log.html) to emphasize the fact that only the *logging* is different in these two cases:
```rust
        Ok(code) => {
            log!(if code == 0 { Level::Info } else { Level::Error },
                 "Launcher exiting with code {}",
                 code);
            process::exit(code);
        }
```
or, if you like less nesting:
```rust
        Ok(code) => {
            let level = if code == 0 { Level::Info } else { Level::Error };
            log!(level, "Launcher exiting with code {}", code);
            process::exit(code);
        }
```